scripts/utilities.py

```diff
@@ -22,9 +24,9 @@ def parse_file(file_name):
     def connect_device():
         """Force connection to device and forward ports, just in case.
         """
-        os.popen("adb kill-server")
-        os.popen("adb devices")
-        os.popen("adb forward tcp:2828 tcp:2828")
+        os.popen("adb kill-server >> /dev/null 2> /dev/null")
```

Comment:
If redirecting to /dev/null, no need to append. I would just write:

   adb kill-server > /dev/null 2>&1

And the same for the other commands.